server/controllers/redis.js

```diff
@@ -40,9 +36,17 @@ exports.getCached = asyncHandler( async (req, res, next) => {
 
 // function used in endpoints to set results to cache
 exports.setCache = async (key, value) => {
-  await setter(key, value).catch(err => { throw err });
+  const keyValue = parseInt(key);
+  // naive implementation of key control
+  if(9000000 <= keyValue && keyValue <= 9350000) {
```

Comment:
This is storing the first 35% of the last 10% of your database, right? Does it matter which 30-40% of the last 10% of the database we store in cache? For example, one could also store product Id numbers 9,600,000 through 10,000,000 (the last 40% of the last 10%). Are there tradeoffs to doing this?

Reply:
there's no benefits I can see aside from being able to predict very easily what productIds would be cached. it was just a simple way to control the keys that would be cached since I don't have any other metric or system in place to designate it